### PR TITLE
overloaded writeModelsToDisk

### PR DESCRIPTION
**Pull request by qiangning**
writeModelsToDisk now can also take user specified dir and model name as inputs.

### CODE REVIEW

File: chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java [hunk 1]
```diff
@@ -90,7 +90,10 @@ public void writeModelsToDisk() {
         chunker.save();
         System.out.println("Done training, models are in " + rm.getString("modelDirPath"));
     }
-
+    public void writeModelsToDisk(String dir, String modelName){
+        chunker.write(dir + modelName + ".lc", dir + modelName + ".lex");
```
Comment (danyaljj) [inline chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java]: Minor: you can be safe and make sure there is [already a folder. ](https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/blob/master/core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/io/IOUtils.java#L58)

Comment (danyaljj) [inline chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java]: And instead of `/` use `File.separator`.

Comment (qiangning) [inline chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java]: Thanks a lot @danyaljj ! Do I need to modify my code accordingly and send another PR?

Comment (danyaljj) [inline chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java]: Just update ur code; it will be reflected here.

Comment (danyaljj) [inline chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java]: Semaphore is angry. Invited you to Semaphore account. Go check it out and see why it's failing.
